src/index.html

```diff
@@ -22,6 +22,362 @@
+            </div>
+            <p class="product-card__reviews">Reviews: 5</p>
+          </div>
+          <div class="product-card__price">
+            <span class="product-card__price-label">Price:</span>
+            <span class="product-card__price-value">$2,199</span>
+          </div>
+          <button
+            data-qa="card-hover"
+            class="product-card__buy-button"
+          >
+            BUY
+          </button>
+        </div>
+        <div
+          data-qa="card"
+          class="product-card"
+        >
+          <img
+            class="product-card__img"
+            src="/src/images/imac.jpeg"
+            alt="imac"
+          />
+          <h2 class="product-card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product-card__code">Product code: 195434</p>
+          <div class="product-card__review">
+            <div class="product-card__stars">
+              <div class="product-card__stars-star"></div>
+              <div class="product-card__stars-star"></div>
+              <div class="product-card__stars-star"></div>
+              <div class="product-card__stars-star"></div>
+              <div class="product-card__stars-star"></div>
+            </div>
+            <p class="product-card__reviews">Reviews: 5</p>
+          </div>
+          <div class="product-card__price">
+            <span class="product-card__price-label">Price:</span>
+            <span class="product-card__price-value">$2,199</span>
+          </div>
+          <button
+            data-qa="card-hover"
+            class="product-card__buy-button"
+          >
+            BUY
+          </button>
+        </div>
+        <div
+          data-qa="card"
+          class="product-card"
+        >
+          <img
+            class="product-card__img"
+            src="/src/images/imac.jpeg"
+            alt="imac"
+          />
+          <h2 class="product-card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product-card__code">Product code: 195434</p>
+          <div class="product-card__review">
+            <div class="product-card__stars">
+              <div class="product-card__stars-star"></div>
+              <div class="product-card__stars-star"></div>
+              <div class="product-card__stars-star"></div>
+              <div class="product-card__stars-star"></div>
+              <div class="product-card__stars-star"></div>
+            </div>
+            <p class="product-card__reviews">Reviews: 5</p>
+          </div>
+          <div class="product-card__price">
+            <span class="product-card__price-label">Price:</span>
+            <span class="product-card__price-value">$2,199</span>
+          </div>
+          <button
+            data-qa="card-hover"
+            class="product-card__buy-button"
+          >
+            BUY
+          </button>
+        </div>
```

Comment:
I noticed that you have a lot of repetitive code here. Consider using a `for` loop to generate these product card elements dynamically. This would make your code more DRY (Don't Repeat Yourself), easier to read and maintain.